src/objects/conf/dev.py

```diff
@@ -21,6 +22,7 @@
 #
 
 DEBUG = True
+IS_HTTPS = False
```

Comment:
I could remove it, but then it would be required to configure it in your envvars (if you want to connect to a notifications API via http). I explicitly added it, in order to let it function similarly to Open Zaak: https://github.com/open-zaak/open-zaak/blob/d5cddbbe2a3f11159852c838455b42755baf7373/src/openzaak/conf/dev.py#L14